include/ioh/problem/pbo/labs.hpp

```diff
@@ -0,0 +1,68 @@
+#pragma once
+#include "pbo_problem.hpp"
+
+namespace ioh
+{
+    namespace problem
+    {
+        namespace pbo
+        {
+            class LABS : public PBOProblem<LABS>
+            {
+            protected:
+                static double correlation(const std::vector<int> x, const int n, int k)
+                {
+                    int x1, x2;
+                    auto result = 0.0;
+                    for (auto i = 0; i < n - k; ++i)
```

Comment:
Also this can be shortened much: `for (auto i = 0; i < n - k; ++i) {result += (x.at(i) * 2 -1) * (x.at(i+k) * 2 -1) }` (assuming max xi=1)